components/profile/create/stages/Form.vue

```diff
@@ -140,19 +147,23 @@ const socialLinks = [
   },
 ]
 
-watch(hasProfile, (hasProfile) => {
-  if (hasProfile) {
-    form.name = userProfile.value?.name ?? ''
-    form.description = userProfile.value?.description ?? ''
-    form.farcasterHandle = userProfile.value?.socials.find(
-      (social) => social.platform === 'Farcaster',
-    )?.handle
-    form.twitterHandle = userProfile.value?.socials.find(
-      (social) => social.platform === 'Twitter',
-    )?.handle
-    form.website = userProfile.value?.socials.find(
-      (social) => social.platform === 'Website',
-    )?.handle
-  }
-})
+watch(
+  hasProfile,
+  (hasProfile) => {
+    if (hasProfile) {
+      form.name = userProfile.value?.name ?? ''
+      form.description = userProfile.value?.description ?? ''
+      form.farcasterHandle = userProfile.value?.socials.find(
+        (social) => social.platform === 'Farcaster',
+      )?.handle
+      form.twitterHandle = userProfile.value?.socials.find(
+        (social) => social.platform === 'Twitter',
+      )?.handle
+      form.website = userProfile.value?.socials.find(
+        (social) => social.platform === 'Website',
+      )?.handle
+    }
+  },
+  { immediate: true },
```

Comment:
Nitpick: `watchEffect` could help simplify this

Reply:
changed to `watchEffect`

thanks :)